src/frontend/src/components/panels/PanelGroup.css.ts

```diff
@@ -0,0 +1,10 @@
+import { style } from '@vanilla-extract/css';
+import { vars } from '../../theme';
+
+export const selectedPanelTab = style({
+  selectors: {
+    '&[data-active]': {
+      background: vars.colors.blue.light
```

Comment:
A: could we use the primary color here?

Reply:
Sure, totally forgot that we support multiple primary colors.